go.mod

```diff
@@ -4,6 +4,7 @@ go 1.18
 
 require (
 	github.com/confluentinc/confluent-kafka-go v1.8.2
+	github.com/ethereum/go-ethereum v1.10.11
```

Comment:
Why is ethereum leaking in here?

Reply:
It's used for ABI encoding to create the report

Reply:
This repo and these interfaces are meant to remain chain-agnostic. If this version is still EVM specific then it is not ready to move out of the core repo yet.

Reply:
If the dependency is limited to the _implementations_, that is fine, but those pieces aren't supposed to be in this repo.

Reply:
Mercury works a bit differently, since this actually _is_ chain agnostic on the transmitter side. The eventual implementation will encode multiple different reports for different types of chain.

Reply:
Median is not an example to be followed in this case. Chain specific implementations of the various components need to be in chain specific repos (or core in the case of EVM).

Reply:
Moved it into core

Reply:
I would second what jordan said - ideally we don't bring geth into this repo. iirc, in the past it has caused a lot of dependency bloat and also some issues with keeping it in sync across testing/core/integrations repos 😬 

I think in the past we've duplicated some simple logic in this repo - and it seems like the only thing used right now is the hash type which is a `[length]byte` implementation + a utility to convert `[]byte` to `common.Hash`

Reply:
Go-ethereum is now completely expunged